<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionEditorDialog</class>
 <widget class="QDialog" name="ConnectionEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VPN Connections</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/L2tpIPsecVpn.qrc">
    <normaloff>:/images/connectEstablished</normaloff>:/images/connectEstablished</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="m_pGridLayout">
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <item row="1" column="1">
    <widget class="QPushButton" name="m_pAdd">
     <property name="text">
      <string>&amp;Add ...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="m_pEdit">
     <property name="text">
      <string>&amp;Edit ...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_pDelete">
     <property name="text">
      <string>&amp;Delete ...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QTableView" name="m_pConnections">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCommandLinkButton" name="m_pPreferencesCommandLinkButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Preferences ...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_pButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/L2tpIPsecVpn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pConnections</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>m_pEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
